src/PaymentExporter.php

```diff
@@ -7,6 +7,15 @@
  */
 class PaymentExporter {
 
+  const INTERVAL_UNITS = [
+    'monthly' => [1, 'M'],
+    'yearly' => [1, 'Y'],
+    'quarterly' => [3, 'M'],
+  ];
```

Comment:
If we can trust the documentation of the `payment_recurrence` module, the payment methods implementing it support `weekly`, `monthly`, `yearly` (and `dayly` for Stripe). 
1. Though not very common, I think campaignion_logcrm should support `W` (and maybe `D`) as well
2. Mentioning ISO 8601 somewhere would make it easier to look this up ;)
3. I don't think we need to expect `quaterly` (instead of `3 monthly`), please prove me wrong
4. It might be dangerous to silently fall back to one-off payments when we get an unexpected interval → at least log a warning, or pass the unmodified value to logcrm?

Reply:
Ok, I’ve dropped `quarterly` which allowed me to simplify the logic a bit. I also introduces `daily` and `weekly`. Neither payment_recurrence nor stripe_payment document which values are actually allowed. :see_no_evil:

Reply:
Thanks! :heart_decoration: 

Yes, I looked it up in the [Stripe API reference](https://stripe.com/docs/api/plans/create#create_plan-interval). The way `stripe_payment` is implemented, it would send ["dai" for "daily"](https://github.com/moreonion/stripe_payment/blob/7.x-2.x/src/Utils.php#L126) causing an API error, but I'd regard that as a bug in `stripe_payment`, let's stick to proper spelling.